common/src/node.rs

```diff
@@ -173,7 +267,18 @@ impl VoxelData {
         match *self {
             VoxelData::Dense(ref mut d) => d,
             VoxelData::Solid(mat) => {
-                *self = VoxelData::Dense(vec![mat; (usize::from(dimension) + 2).pow(3)].into());
+                let lwm = usize::from(dimension) + 2;
+                let mut data = vec![Material::Void; lwm.pow(3)];
+
+                // Populate all blocks except the margins, as margins are not fully implemented yet.
```

Comment:
What does it mean for the margin to be "not fully implemented", and why is populating it with void the solution?

Reply:
In worldgen, all dense chunks have no margin. I'm essentially saying that this is a TODO for #41.